projects/core/src/organization/services/organization-meta.resolver.ts

```diff
@@ -88,12 +48,12 @@ export class OrganizationMetaResolver extends PageMetaResolver
 
   protected resolveBreadcrumbData(
```

Comment:
I don't think we need all this complexity honestly. I'm mostly worried about wiring the route to the localisation, as customers should be able to reconfig the route, and that would force them to recreate the localizations as well. Given that the localization branch 'organization' is used both by components and this meta resolver, it's getting nasty. 

I'd rather simplify, and do this: 
- use the super to bring in the home path
- use a static link to the organization label for the breadcrumb. This page resolver is explicitly used for the `CompanyPageTemplate` page template, so there's no reason to connect it to the routes actually.